### ISSUE TITLE
Add an ability to delete products and/or product documentation

### ISSUE THREAD

**Issue by frankkilcommins** [frankkilcommins/SwaggerHub-Portal-Management#11]
It's also needed to be able to also DELETE product documentation as part of the automation.

By convention it makes sense that if a product is managed by automation, then it's _fully_ managed by automation. Thus before starting to create or update existing documentation it would make sense to delete any `table-of-content` entries that are not defined within the `manifest.json`

Additionally, there might be a desire to have all portal products falling under the management of the automation pipelines, thus a new metadata file for root level OR an additional environment variable at the repo level could be used to control the deletion of products not part of the products folder.
